controllers/storageconfig_controller.go

```diff
@@ -59,8 +60,10 @@ func newStorageSecret(dataConnectionSecretsList *corev1.SecretList) *corev1.Secr
 		dataConnectionElement["default_bucket"] = string(secret.Data["AWS_S3_BUCKET"])
 		dataConnectionElement["bucket"] = string(secret.Data["AWS_S3_BUCKET"])
 		dataConnectionElement["region"] = string(secret.Data["AWS_DEFAULT_REGION"])
-		if secret.Data["AWS_CA_BUNDLE"] != nil {
-			dataConnectionElement["certificate"] = string(secret.Data["AWS_CA_BUNDLE"])
+
+		if odhCustomCertData != "" {
+			dataConnectionElement["certificate"] = odhCustomCertData
+			dataConnectionElement["cabundle_configmap"] = constants.KServeCACertConfigMapName
```

Comment:
Is this a breaking change compared to previous behaviour? What if a user already has a secret with `AWS_CA_BUNDLE`?

Reply:
The existing code was for a self-signed certificate in ModelMesh, but it was not officially supported because the UI was not implemented. Therefore, replacing the code has no impact.